nightfall/api.py

```diff
@@ -81,22 +86,26 @@ def scan_text(self, texts: List[str], detection_rules: Optional[List[DetectionRu
         :returns: list of findings, list of redacted input texts
         """
 
-        if not detection_rule_uuids and not detection_rules:
-            raise NightfallUserError("at least one of detection_rule_uuids or detection_rules required", 40001)
+        if not policy_uuids and not detection_rule_uuids and not detection_rules:
```

Comment:
do you want to add a test for this case too?

Reply:
looks like it's already there, just renamed the test for visibility